src/providers/bank.service.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {Injectable} from '@angular/core';
+import 'rxjs/add/operator/map';
+import {environment} from "../environments/environment";
+import {BankTransferService} from './bank.service';
+import {AuthHttp} from "angular2-jwt";
+/**
+ * Bank test suite
+ */
+describe('Bank Service', () => {
+    beforeEach(() => {
+        this.BankTransferService = new BankTransferService();
```

Comment:
I seem to be getting compilation errors here, I believe it is non-default constructor.